proto/validator/v1/validator.proto

```diff
@@ -137,7 +131,10 @@ message Deposit {
 }
 
 message GetCtipRequest {
-  uint32 sidechain_number = 1;
+  // Sidechain number to fetch chain tip for. This argument is in fact
+  // NOT optional! It is marked as such to clearly differentiate it
+  // from passing in 0, which is a valid sidechain number.
+  optional uint32 sidechain_number = 1;
```

Comment:
If this value is not optional, consider using `UInt32Value` https://protobuf.dev/reference/protobuf/google.protobuf/#uint32-value

Reply:
Fixed!